e2e_tests/tests/task/test_generic_tasks.py

```diff
@@ -231,8 +232,16 @@ def test_pause_and_unpause_generic_task() -> None:
 
     detproc.check_call(sess, command)
 
-    pause_resp = bindings.get_GetTask(sess, taskId=task_resp.taskId)
-    assert pause_resp.task.taskState == bindings.v1GenericTaskState.PAUSED
+    # The task may still be PAUSING, retry a few times.
+    retries = 5
```

Comment:
nit: We could use `task.wait_for_task_state()`, but this solution is fine

Reply:
oh didn't see this, great! thanks!